K2Field.K2NE.ServiceBroker/ServiceObjects/EmailTemplate/SmoClientHelper.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using SourceCode.SmartObjects.Client;
+
+namespace K2Field.K2NE.ServiceBroker.ServiceObjects.EmailTemplate
+{
+    public class SmoClientHelper : SmartObjectClientServer, IDisposable
```

Comment:
Basically, this class can be avoided. We can simply open the connection and pass the SmoClientServer, opened in a standard way. I thought of enhancing the class, but finally implemented this in a different way.